src/providers/ExtensionDetectionProvider/extensionDetection.spec.ts

```
@@ -0,0 +1,660 @@
+  pocketUniverseDatasetCsnDetector,
+  pocketUniverseHtmlDataCsnSnapshotDetector,
+  postMessageProxyDetector,
+  resourceFetchDetector,
+  runDetectors,
+  stylesheetDetector,
+  withTimeout,
+  type ExtensionDefinition,
+} from './utils';
+
+const runInlineScript = (source: string): void => {
+  globalThis.eval(source);
```

Biome (2.4.15)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)